fw/beef.cpp

```diff
@@ -322,7 +317,8 @@ void process_combos(config* current_config,
     ignore_combo = false;
     timer_init(combo_timer);
     timer_init(combo_lights_timer);
-    timer_init(combo_tt_led_timer);
+    timer_init(&RgbManager::Turntable::combo_timer);
+    timer_init(&RgbManager::Bar::combo_timer);
```

Comment:
Why do we need separate timers for each combo? Is there ever a case when more than one timer will be running at once?

Reply:
The rationale was that there might be a combo that would apply an effect to the bar, but I guess we can remove it and just re-add it when it's needed.